www/install/php/Update-22.04.0-beta.1.php

```diff
@@ -38,6 +39,11 @@
             AFTER `contact_comment`"
         );
     }
+
+    $errorMessage  = 'Unable to delete logger entry in cb_tag';
+    $statement = $pearDB->query("DELETE FROM cb_tag WHERE tagname = 'logger'");
+
+    $pearDB->commit();
```

Comment:
you can remove transactions instructions
cause there will bea PDOException cause you are committing but you are not in a transaction (since ADD COLUMN add an implicit commit)